lib/PuppeteerSharp.Tests/FrameTests/FrameManagementTests.cs

```diff
@@ -124,9 +124,9 @@ public async Task ShouldReportFrameName()
                 return new Promise(x => frame.onload = x);
             }", TestConstants.EmptyPage);
 
-            Assert.Equal(string.Empty, Page.Frames.ElementAt(0).Name);
-            Assert.Equal("theFrameId", Page.Frames.ElementAt(1).Name);
-            Assert.Equal("theFrameName", Page.Frames.ElementAt(2).Name);
+            Assert.Single(Page.Frames.Where(f => f.Name == string.Empty));
+            Assert.Single(Page.Frames.Where(f => f.Name == "theFrameId"));
+            Assert.Single(Page.Frames.Where(f => f.Name == "theFrameName"));
```

Comment:
use `Assert.Single(Page.Frames, frame => frame.Name == "theFrameName");`
do this anywhere 
```cs
Assert.Single(enumerable.Where(predicate));
```
is used